src/explainer_prep.R

```diff
@@ -1,112 +1,220 @@
+      height = 0.06,
+      width = 0.06,
+      hjust = 1,
+      vjust = 0.5
+    ) +
+    draw_label(
+      low_lab,
+      x = 0.754,
+      y = 0.37,
+      size = 22,
+      hjust = 0.5,
+      vjust = 0,
+      fontfamily = font_legend,
+      color = low_col
+    ) +
+    draw_plot(
+      low_range_arrow,
+      x = 0.72,
+      y = 0.395,
+      height = 0.06,
+      width = 0.06,
+      hjust = 1,
+      vjust = 0.5
+    ) + 
+  # add legend
+  draw_plot(restyle_legend,
+            x = (1-plot_margin)*0.5,
+            y = 0.133,
+            height = 0.12 ,
+            width = 0.02-plot_margin) +
+    # draw title
+    draw_label(sprintf('%s %s', plot_month, plot_year),
+               x = plot_margin*7.5, y = 1-plot_margin*2.5,
+               size = 72,
+               hjust = 0,
+               vjust = 1,
                fontfamily = font_legend,
-               color = low_col) +
-    draw_label(high_lab,
-               x = 0.86, y = 0.76, 
-               size = 5.5, 
-               hjust = 0.5, vjust = 1,
+               color = text_color,
+               lineheight = 1)  +
+    # stylized streamflow title
+    draw_image(title_flow,
+               x = plot_margin*7.6,
+               y = 1-(2.4*plot_margin),
+               height = 0.2,
+               width = 0.76,
+               hjust = 0,
+               vjust = 1) +
+    # percentile info
+    draw_label(flow_label,
+               x = (1-plot_margin)*0.246,
+               y = 0.27,
+               hjust = 0,
+               vjust = 1,
                fontfamily = font_legend,
-               color = high_col) +
-    draw_label(typ_lab,
-               x = 0.875, y = typ_lab_ypos, 
-               size = 5.5, 
-               hjust = 0.5, vjust = 1,
+               color = text_color,
+               size = 24) +
+    # add data source
+    draw_label(source_label, 
+               x = plot_margin + 0.8, y = plot_margin*2, 
+               fontface = "italic", 
+               size = 20, 
+               hjust = 1, vjust = 0,
                fontfamily = font_legend,
-               color = text_color) +
-    draw_plot(high_range_arrow, # for high streamflow
-              x = 0.77, y = 0.755,
-              height = 0.035, width = 0.05,
-              hjust = 0, vjust = 0.5) +
-    draw_plot(normal_range_arrow, # for typical streamflow
-              x = 0.77, y = typ_arr_ypos,
-              height = 0.035, width = 0.05,
-              hjust = 0, vjust = 0.5) +
-    draw_plot(low_range_arrow, # for low streamflow
-              x = 0.77, y = 0.35,
-              height = 0.035, width = 0.055,
-              hjust = 0, vjust = 0.5)
+               color = text_color,
+               lineheight = 1.1) +
+    # add logo
+    draw_image(usgs_logo, x = plot_margin + 0.14, y = plot_margin*1.6,
+               width = 0.135, hjust = 0, vjust = 0, halign = 0, valign = 0)
+  
+  # overlay IG grid 
+  if (check_ig_safezone) {
+    g <- g + draw_grob(ig_grid_raster)
+  }
```

Comment:
SC: Consider adding this functionality to `national_ig()` as well, since labels + legend will be added there